### PR TITLE
AutoPlugin: support performance Hint

### PR DESCRIPTION
**Pull request by tiger100256-hu**
Signed-off-by: Hu, Yuan2 <yuan2.hu@intel.com>

### Details:
 - *open the configure gate of auto plugin in benchmark_app*
 - *check performance hint related function test of auto plugin*

### Tickets:
 - *55563*

### CODE REVIEW

**Comment by myshevts** [comment]
pls rebase to master (that now have a fix on the Auto tests crash- that exactly happened for your pre-commit) and rerun the CI

File: inference-engine/src/multi_device/multi_device_plugin.cpp [hunk 1]
```diff
@@ -268,7 +266,7 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
                  }
              }
              iter->config = deviceConfig;
-             strDevices = iter->deviceName;
+             strDevices += iter->deviceName;
```
Comment (tiger100256-hu) [inline inference-engine/src/multi_device/multi_device_plugin.cpp:269]: @myshevts  fix an error here

**Review by myshevts — APPROVED** [review]

File: inference-engine/src/multi_device/multi_device_plugin.cpp (hunk 1 above)
Comment (myshevts) [reply inference-engine/src/multi_device/multi_device_plugin.cpp:269]: likely you need another PR

Comment (peterchen-intel) [reply inference-engine/src/multi_device/multi_device_plugin.cpp:269]: @myshevts What does it mean "another PR"? If anything is missed for https://jira.devtools.intel.com/browse/CVS-55563?

**Pull request merged**